crates/autopilot/src/solvable_orders.rs

```diff
@@ -604,39 +611,39 @@ async fn update_task(
     }
 }
 
-fn get_orders_with_native_prices(
+async fn get_orders_with_native_prices(
     orders: Vec<Order>,
     native_price_estimator: &CachingNativePriceEstimator,
     metrics: &Metrics,
+    additional_tokens: Vec<H160>,
```

Comment:
nit: That way you don't have to collect the tokens into a vector
```suggestion
    additional_tokens: impl IntoIterator<Item = H160>,
```